.github/assert-deployed-bytecode.js

```diff
@@ -0,0 +1,104 @@
+const { AssertionError } = require('chai')
+const chalk = require('chalk')
+const { web3 } = require('hardhat')
+const { readJSON } = require('../scripts/helpers/fs')
+const { APPS_TO_NAMES, CONTRACTS_TO_NAMES, IGNORE_METADATA_CONTRACTS } = require('./deployed-bytecode-consts')
+
+function compareSymbolWise(first, second, ignoreMetadata) {
```

Comment:
The function name may be misleading to the reader a bit.
I suggest using smth like `isBytecodeSimilar`

Reply:
fixed